$common/runfile.py

```diff
@@ -76,7 +79,9 @@ def main():
     result = helper.run_command(compile_cmd)
 
     # handle compilation errors
-    feedback.compilation_feedback(result)
+    # For teaching assistant that test some tasks might want the full feedback
+    full_log = username == config["username_for_full_log"]

```

Comment:
Should be `feedback_settings` instead of `config` here? With respect to previous coment `username in `feedback_settings["username_for_full_log"]`

Reply:
Typo : thanks. I will change it directly